benchmarks/static_set/contains_bench.cu

```diff
@@ -41,7 +41,7 @@ void static_set_contains(nvbench::state& state, nvbench::type_list<Key, Dist>)
 
   thrust::device_vector<Key> keys(num_keys);
 
-  key_generator gen{};
+  [[maybe_unused]] key_generator gen{};
   gen.generate(dist_from_state<Dist>(state), keys.begin(), keys.end());
```

Comment:
How can `gen` be `maybe_unused` when it is used immediately in the following line? What am I missing?